.github/workflows/build.yml

```diff
@@ -24,6 +24,8 @@ jobs:
       STAGING_CLOUDFRONT_DISTRIBUTION_ID: E2ELTBTA2OFPY2
       REVIEW_CLOUDFRONT_DISTRIBUTION_ID: E3267W09ZJHQG9
       VITE_FOUNDATION_BUILD: ${{ github.repository_owner == 'microbit-foundation' }}
+      # Optionally disable the PWA config (service worker and web manifest)
+      # DISABLE_PWA: '1'
```

Comment:
Let's confirm what the default behaviour should be.

Reply:
I've flipped it to enable only for Foundation builds so that folks (especially students) don't get bitten by the SW when doing more experimental work.